walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +123,64 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeIsConsideredWithWholeDayOptionalAttendee() {
```

Comment:
Try adding a test with your timerange in non chronological order i guarantee your algo fails

Reply:
Added `everyAttendeeIsConsideredWithWholeDayOptionalAttendeeWithNonChronologicalEvents()` and passes:)